src/components/gallery.tsx

```diff
@@ -65,7 +65,11 @@ export default class Gallery extends React.Component<Props> {
     let { items, onKeyDown } = this.props
 
     return (
-      <TransitionGroup className="ars-gallery" onKeyDown={onKeyDown}>
+      <TransitionGroup
```

Comment:
Lol, you got to this before I could leave my review.

Reply:
😎